src/otx/core/utils/build.py

```diff
@@ -40,6 +40,16 @@ def build_mm_model(config: DictConfig, model_registry: Registry, load_from: str
     return model
 
 
+def get_default_async_reqs_num() -> int:
+    """Returns a default number of infer request for OV models."""
+    import os
+
+    reqs_num = os.cpu_count()
+    if reqs_num is not None:
+        return max(1, int(reqs_num / 2))
+    return 1
```

Comment:
Can we notify this behavior to an user such as
```suggestion
    reqs_num = os.cpu_count()
    reqs_num = max(1, int(reqs_num / 2)) if reqs_num is not None else 1
    msg = f"Set the number of OpenVINO inference requests to {reqs_num} because you did not configure 'max_num_requests'"
    warnings.warn(msg)
```

Reply:
We can change it similar to this, yes

Reply:
[done](https://github.com/openvinotoolkit/training_extensions/pull/2769/commits/1da85964b12868bb793297fab408ea395fe88500)